projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -12,12 +12,11 @@
  */
 function wpcom_add_wpcom_menu_item() {
 	if ( function_exists( 'wpcom_is_nav_redesign_enabled' ) && wpcom_is_nav_redesign_enabled() ) {
-		$domain = wp_parse_url( home_url(), PHP_URL_HOST );
 		add_menu_page(
-			esc_attr__( 'WordPress.com', 'jetpack-mu-wpcom' ),
-			esc_attr__( 'WordPress.com', 'jetpack-mu-wpcom' ),
+			esc_attr__( 'All Sites', 'jetpack-mu-wpcom' ),
```

Comment:
Super nit: Should this be `All Sites` or `All sites`? 

On Calypso (nav unification), we're using sentence case:

<img width="235" alt="image" src="https://github.com/Automattic/jetpack/assets/1525580/a05f17bd-e856-4bfc-860e-667e5cc8ae63">

<img width="254" alt="image" src="https://github.com/Automattic/jetpack/assets/1525580/c9971545-6bf5-4fbe-b8fe-8e5f85d164f3">

Reply:
@nuriapenya @lucasmendes-design thoughts? Seems like we have a mix of both at the moment:

![CleanShot 2024-02-25 at 05 28 44@2x](https://github.com/Automattic/jetpack/assets/5634774/d9960650-84c5-4b48-bdf9-340baf9a4b02)